auth-server/test/resolvers/mutation/addressLinkStart.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { expect } from 'chai';
+import { AuthenticationError } from 'apollo-server';
+import 'mocha';
+
+import User from '../../../src/model/User';
+import Address from '../../../src/model/Address';
+import addressLinkStart from '../../../src/resolvers/mutation/addressLinkStart';
+import signup from '../../../src/resolvers/mutation/signup';
+import { Context } from '../../../src/types';
+import messages from '../../../src/utils/messages';
+
+describe('addressLinkStart mutation', () => {
+	let signupResult;
+	const fakectx: Context = {
+		req: {
+			headers: {}
+		},
+		res: {
+			cookie: () => {}
+		}
+	} as any;
+	const email = 'test@email.com';
+	const password = 'testpass';
+	const username = 'testuser';
+	const name = 'test name';
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx);
+		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+	});
+
+	after(async () => {
+		await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.del();
+	});
+
+	it('should be able to start linking address', async () => {
+		const network = 'kasuma';
+		const address = '5CRvXUKwLDzFM1u9reZ1quf14pNbsw27EZWCBxPMmpydBfYV';
```

Comment:
We should use Alice here as well.

Reply:
When using Alice address the signature verification is giving false for signatue generated by signer tool